### DISCUSSION TITLE
How to use?

### DISCUSSION THREAD

**Discussion by Guo8a** [google/flutter-mediapipe#3] [Q&A] [answered]
I used to use the tflite_flutter_helper library, but it has now been deprecated and it is recommended to use the MediaPipe library instead. How do I use this library? Are there any sample codes available? Why haven't I found any code in this repository?

**Comment by PaulTR** [comment, marked as answer]
Yes, it is currently empty. Per discussions in flutter-tflite, this will be built over the next few months. You can continue to use the helper plugin until then, but it will not be receiving updates.

**Reply by Guo8a** [reply]
Okay, thank you to the flutter-mediapipe team.
I hope that a plugin supporting flutter 3.10.0 and compatible with tflite_flutter on both iOS and Android platforms can be developed as soon as possible.
Thank you for your hard work, and best wishes to you all!

**Reply by manofgodjr** [reply]
This conversation was helpful to me

**Reply by manofgodjr** [reply]
> Yes, it is currently empty. Per discussions in flutter-tflite, this will be built over the next few months. You can continue to use the helper plugin until then, but it will not be receiving updates.

What is the helper plugin

**Reply by Guo8a** [reply]
https://pub.dev/packages/tflite_flutter_helper
